cmd/vic-machine/common/dns.go

```diff
@@ -0,0 +1,56 @@
+package common
```

Comment:
This file is missing the copyright header - see `cmd/vic-machine/configure/configure.go` for example.